sbncode/BeamSpillInfoRetriever/SBNDBNBZEROBIASRetriever/MWRData.cpp

```diff
@@ -0,0 +1,71 @@
+#include <iostream>
```

Comment:
I would definitely move this code up and reference it in one place so we don't have to maintain the copies